Header header

#Массив значений короткого замыкания на ключах системы
bool[16] status